Venmo Universal/App Links (#911)
braintree/braintree_android: CHANGELOG.md

# Braintree Android SDK Release Notes

## unreleased

* Venmo
  * Add `setFallbackToWeb()` to `VenmoRequest`
    * If set to `true` customers will fallback to a web based Venmo flow if the Venmo app is not installed

>     * This method uses App Links and is more secure than the Deep Link based flow

Author: ```suggestion
    * This method uses App Links instead of Deep Links
```
